pyproject.toml

```diff
@@ -187,13 +291,21 @@ ignore = [
 ]
 
 
+# ======================================================================================
```

Comment:
This section can be safely removed, right? All done by ruff if I am not mistaken.